homework_10/CurrentTime.java

```diff
@@ -0,0 +1,11 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class CurrentTime {
```

Comment:
Rename `CurrentTime` to `DateTimeUtils`, make this class final, and add a default private constructor. This is the way how the utility classes should be declared.